npm-shrinkwrap.json

```diff
@@ -14910,6 +15447,17 @@
         }
       }
     },
+    "winusb-driver-generator": {
```

Comment:
fsevents and winusb-driver-generator shouldn't be in teh shrinkwrap file.

Reply:
I don't even know how they got there, I'm on OSX